### PR TITLE
Feat/keyp 272 build examples

### PR DESCRIPTION
**Pull request by odelcroi**
- Build webservice example based on Quarkus

- Build websocket example based on Quarkus

- Added inside the plugin folder waiting for the reordering of the example project

### CODE REVIEW

**Review by andrei-cristea — CHANGES_REQUESTED** [review]

File: java/component/keyple-plugin/remote-se/example/common/src/main/java/org/eclipse/keyple/remotese/example/app/RemoteSePluginObserver.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+   *
+   * @param virtualReader the virtual reader on where to execute the business logic
+   * @return output object
+   */
+  private Object executeService(RemoteSeServerReader virtualReader) {
+    logger.info("Executing ServiceId : {}", virtualReader.getServiceId());
+
+    // EXECUTE_CALYPSO_SESSION_FROM_REMOTE_SELECTION
+    if ("EXECUTE_CALYPSO_SESSION_FROM_REMOTE_SELECTION".equals(virtualReader.getServiceId())) {
+      UserInfo userInput = virtualReader.getUserInputData(UserInfo.class);
```
[the remark this answers is not in the document]
Comment (odelcroi) [reply java/component/keyple-plugin/remote-se/example/common/src/main/java/org/eclipse/keyple/remotese/example/app/RemoteSePluginObserver.java:84]: ok

File: java/component/keyple-plugin/remote-se/example/keyple-example-remotese-web-service/gradlew [hunk 2]
```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```
[the remark this answers is not in the document]
Comment (odelcroi) [reply java/component/keyple-plugin/remote-se/example/keyple-example-remotese-web-service/gradlew:1]: yes, it is advised to commit the gradle wrapper

File: java/component/keyple-plugin/remote-se/example/keyple-example-remotese-web-service/gradlew.bat [hunk 3]
```diff
@@ -0,0 +1,89 @@
+@rem
```
[the remark this answers is not in the document]
Comment (odelcroi) [reply java/component/keyple-plugin/remote-se/example/keyple-example-remotese-web-service/gradlew.bat:1]: yes, it is advised to commit the gradle wrapper

File: java/component/keyple-plugin/remote-se/example/common/src/main/java/org/eclipse/keyple/remotese/example/app/RemoteSePluginObserver.java [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  public void update(PluginEvent event) {
+    logger.info(
+        "Event received {} {} {}",
+        event.getEventType(),
+        event.getPluginName(),
+        event.getReaderNames().first());
+
+    switch (event.getEventType()) {
+      case READER_CONNECTED:
+        // retrieve serviceId from reader
```
[the remark this answers is not in the document]
Comment (odelcroi) [reply java/component/keyple-plugin/remote-se/example/common/src/main/java/org/eclipse/keyple/remotese/example/app/RemoteSePluginObserver.java:57]: ok

File: java/component/keyple-plugin/remote-se/example/common/src/main/java/org/eclipse/keyple/remotese/example/app/RemoteSePluginObserver.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (odelcroi) [reply java/component/keyple-plugin/remote-se/example/common/src/main/java/org/eclipse/keyple/remotese/example/app/RemoteSePluginObserver.java:84]: ok

**Review by andrei-cristea — APPROVED** [review]

**Pull request merged**